Sources/SwiftInspectorVisitors/ExtensionVisitor.swift

```diff
@@ -77,7 +84,9 @@ public final class ExtensionVisitor: SyntaxVisitor {
       let structVisitor = StructVisitor(parentTypeName: extensionInfo.name)
       structVisitor.walk(node)
 
-      structs.append(contentsOf: structVisitor.structs)
+      innerStructs.append(contentsOf: structVisitor.structs)
```

Comment:
Nit: in some places we use `+=` and in others we use `append(...)`. Would be nice to be consistent, so that differences have meaning (which I'm not sure these do).

Reply:
e8531b59c12a28b8f76fd8f7afc1b85b76a3e9e2